tools/binlog/operation.md

```diff
@@ -0,0 +1,128 @@
+---
+title: TiDB-Binlog Cluster operations
+summary: Learn how to operate the cluster version of TiDB-Binlog.
+category: tools
+---
+
+# TiDB-Binlog operations
```

Comment:
Please keep this heading consistent with the `title` in metadata.

Reply:
Updated in https://github.com/pingcap/docs/pull/1078/commits/bf9232fea2dcc1d572c9830225f014a5b84e3908